code/modules/client/preference/preferences_toggles.dm

```diff
@@ -331,3 +331,11 @@
 	prefs.toggles2 ^= PREFTOGGLE_2_DEATHMESSAGE
 	prefs.save_preferences(src)
 	to_chat(src, "You will [(prefs.toggles2 & PREFTOGGLE_2_DEATHMESSAGE) ? "now" : "no longer"] see a notification in deadchat when a player dies.")
+
+/client/verb/toggle_reverb()
```

Comment:
Make absolutely certain this is on, by default, if you have a database.

I don't want the default experience to be "oh, reverb is off, you have to turn it on to get the benefits"--I want the fact we have reverb to be a known thing on para.

We have this issue with some instrument and other prefs---people don't know they exist until they're informed they can turn it on.

Reply:
It is on by default. I have verified.

Reply:
👍